client/src/components/menubar/index.js

```diff
@@ -419,6 +432,21 @@ class MenuBar extends React.Component {
             />
           </Tooltip>
         </div>
+        <Tooltip
```

Comment:
The buttons in the menu bar are not all aligned on the same axis.  See the image below - the centroid "toggle" is at least a pixel lower than the others

![image](https://user-images.githubusercontent.com/3740062/72372344-02ae8f00-36bb-11ea-9aef-2afb6dfa165a.png)

Reply:
Good catch, it's realigned now